compiler/backend/src/org/jetbrains/kotlin/codegen/inline/SMAP.kt

```diff
@@ -72,14 +71,8 @@ class SourceMapCopier(val parent: SourceMapper, private val smap: SMAP, val call
             return mappedLineNumber
         }
 
-        val range = lastVisitedRange?.takeIf { lineNumber in it }
-            ?: smap.findRange(lineNumber)
-            ?: error("Can't find range to map line $lineNumber in ${smap.sourceInfo.source}: ${smap.sourceInfo.pathOrCleanFQN}")
```

Comment:
I'm not sure that this is major, but in anyway: why are you prefer to remove `error`?

Reply:
IMO it's better to accept as much input as possible without throwing internal errors. In particular, this error is why the "synthetic" line numbers have to be under a flag right now even though compilers not aware of their meaning could've simply discarded them.

Reply:
Thank you! cc @yanex